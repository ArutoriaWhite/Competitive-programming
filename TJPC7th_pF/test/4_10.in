11632039 993
5385 0
22207 1
30827 1
62150 0
79560 1
95321 1
111456 1
125709 0
137823 0
170929 0
206619 1
206747 1
209914 0
213905 1
221719 1
225400 1
229170 0
231920 1
233933 0
238382 1
247053 1
250265 0
253997 0
261300 0
263462 1
288232 0
293047 1
296221 0
296709 1
312242 0
330833 0
335292 1
337115 0
351448 0
353319 0
356034 0
356675 0
361727 0
379914 0
392978 1
408759 0
426016 1
449181 1
467426 1
507703 1
511336 1
516387 0
519119 0
534277 1
537634 1
559650 1
570543 0
579804 0
581780 0
587306 0
598225 1
609014 0
621913 0
622486 0
660081 0
665268 0
736220 1
745077 1
745648 1
747861 1
751019 1
755957 1
789659 0
793608 0
819572 1
828158 0
845980 0
856999 0
863635 1
886047 0
888574 1
919769 1
937364 1
946120 0
949080 1
969413 0
990401 1
1001849 0
1005936 0
1009276 1
1015530 1
1037305 0
1039467 0
1044182 1
1047125 0
1051234 0
1065552 1
1075027 1
1079105 1
1084312 1
1098507 0
1129422 0
1140268 0
1140549 1
1142045 1
1149923 1
1155303 1
1166496 1
1179042 1
1183132 0
1190952 1
1208627 1
1231249 1
1232984 0
1234713 1
1238486 1
1247880 0
1250440 1
1266183 1
1323304 0
1325842 1
1335062 0
1345917 0
1364928 1
1396737 1
1410246 1
1415197 0
1418499 1
1436220 1
1442196 0
1467868 0
1475283 1
1480594 0
1495771 0
1511940 1
1513803 0
1527867 0
1531666 1
1539843 1
1541654 0
1542117 1
1549167 1
1555108 1
1576151 0
1587411 0
1615376 1
1638026 0
1643930 1
1647680 1
1676328 0
1690962 0
1704861 1
1706909 0
1731485 1
1751063 0
1754289 0
1770769 1
1781851 1
1794684 0
1798309 0
1814130 0
1821101 1
1826262 1
1852359 1
1894860 0
1929092 0
1931786 0
1936092 1
1940276 0
1952385 1
1961695 0
1989159 1
1999464 0
2018677 1
2044858 0
2045904 1
2064936 1
2073585 1
2078789 0
2089949 0
2095785 1
2103586 1
2108472 1
2111235 0
2158161 1
2164930 0
2187272 0
2188345 1
2192291 1
2216041 1
2244652 0
2257947 0
2258561 0
2261388 1
2264054 1
2283257 0
2307719 0
2308591 1
2311510 0
2319563 0
2342014 1
2345049 1
2348038 0
2351095 1
2352386 0
2352493 1
2357233 0
2357899 1
2383529 1
2402001 1
2412790 1
2420361 0
2452405 1
2457926 0
2462473 1
2465681 1
2470099 1
2472710 1
2474133 1
2478574 0
2495489 0
2496109 0
2497334 0
2499502 1
2500104 1
2538668 0
2551739 1
2552105 0
2553985 1
2571410 0
2574220 1
2601196 0
2614427 1
2632439 0
2639043 0
2639241 0
2653575 0
2666512 0
2695665 1
2713306 0
2716326 0
2716364 1
2720421 0
2738975 1
2743766 0
2781758 0
2788412 1
2791969 1
2793241 1
2836850 1
2848057 1
2858323 1
2861106 0
2872626 1
2887769 1
2892750 1
2932473 1
2955293 0
2968864 1
2979857 1
2989839 0
2991014 1
2991201 1
2992352 1
3012638 0
3018731 1
3023684 1
3028887 0
3051497 0
3057507 1
3091137 0
3097185 1
3101713 1
3110844 1
3130657 1
3133382 0
3140785 1
3140990 1
3154752 1
3172085 0
3180362 0
3184653 1
3185184 1
3192867 0
3193241 1
3196103 0
3200365 0
3201554 0
3232084 0
3239050 0
3254580 0
3259680 0
3262223 1
3309981 0
3314588 0
3335112 1
3354310 1
3368138 1
3379204 1
3383740 0
3396283 0
3468891 0
3492382 1
3509038 1
3511125 0
3534476 1
3563939 1
3564170 1
3577173 0
3581091 1
3626051 0
3682452 1
3684094 0
3684428 0
3696962 0
3732467 0
3737073 0
3745740 0
3747024 1
3747367 1
3757419 0
3770315 1
3772374 1
3774309 0
3781742 1
3817958 1
3827688 1
3885576 1
3886643 1
3894458 0
3912732 0
3916615 1
3921455 1
3922717 1
3928517 1
3928559 1
3938958 1
3972431 1
3976383 1
3985350 0
4010892 0
4023816 0
4038389 1
4045852 1
4047370 1
4053324 1
4055439 1
4105351 0
4116112 1
4117356 1
4121805 1
4128573 1
4137210 1
4143861 0
4152904 0
4172073 1
4181655 1
4222317 0
4242155 0
4249790 0
4252431 1
4256069 0
4259479 1
4274126 1
4292107 1
4297291 1
4298924 1
4325279 0
4333215 0
4339729 0
4339735 0
4398689 1
4405457 0
4407549 0
4416820 0
4461123 0
4462533 0
4473404 1
4480759 1
4485753 1
4511919 0
4522425 0
4523569 0
4532610 1
4542098 0
4542742 1
4548347 0
4561890 0
4594499 1
4597874 1
4605806 0
4628573 0
4630273 1
4632958 0
4642736 0
4656538 1
4659638 0
4695000 1
4698462 0
4719893 0
4735351 0
4737300 1
4759601 0
4784598 1
4804447 1
4806003 0
4810726 1
4833286 0
4834280 0
4836602 0
4847149 0
4874540 1
4874826 0
4891657 1
4905781 0
4907643 1
4915783 0
4916341 0
4917531 1
4931256 1
4939538 1
4949045 0
4970388 1
4979676 0
4990206 1
4994360 1
5007248 1
5023064 1
5025013 0
5039428 1
5071865 0
5072191 1
5072577 1
5090042 0
5096520 0
5099335 0
5127576 0
5136615 1
5155841 0
5170159 0
5174942 1
5175992 0
5226740 1
5237310 1
5238582 0
5247319 0
5250051 1
5286414 0
5298783 1
5313715 0
5320125 0
5344292 1
5345301 0
5378942 0
5381338 1
5403606 1
5409475 0
5412525 1
5413763 0
5454842 0
5471470 0
5474809 1
5484292 1
5487803 1
5514270 0
5519344 1
5525259 1
5525404 1
5528997 0
5538281 1
5558271 1
5559843 0
5562841 0
5574368 1
5576844 0
5592196 1
5603712 0
5621167 0
5625926 1
5626393 0
5626506 1
5632548 1
5642342 1
5642900 1
5647384 1
5669217 0
5684326 1
5713901 0
5715277 1
5750652 0
5798473 0
5813525 0
5816581 0
5825389 0
5838624 0
5839612 1
5851593 1
5852188 1
5874714 1
5883137 0
5883545 0
5890226 1
5910019 1
5914001 1
5929936 0
5940131 1
5952045 1
5962017 0
5973381 1
5975369 1
6010202 1
6018983 1
6029105 1
6047995 1
6054268 1
6069483 1
6078188 1
6081929 0
6081955 1
6086752 0
6093458 1
6110801 0
6114231 0
6123391 1
6132380 1
6137787 0
6140747 1
6142272 0
6156703 0
6167072 1
6168187 1
6173625 1
6187383 1
6189066 0
6194569 0
6197426 0
6220355 0
6221000 1
6245062 0
6256706 1
6264126 1
6269735 0
6272187 1
6298658 1
6302055 1
6323741 1
6337020 0
6360912 0
6363975 0
6370304 1
6373240 0
6385952 1
6389040 0
6404521 1
6413452 1
6447050 0
6464160 0
6465062 1
6466719 0
6480545 0
6510263 0
6518450 1
6535856 1
6539445 1
6540228 1
6600169 1
6614272 0
6625806 1
6629101 0
6644681 0
6648311 0
6661761 1
6667180 1
6677397 0
6691196 0
6692789 1
6695434 0
6709214 0
6746710 1
6765609 1
6768700 1
6772970 0
6793192 1
6800991 0
6812732 0
6817102 1
6820994 0
6864287 1
6881975 0
6905483 0
6908822 1
6915183 0
6920937 1
6926913 0
6928909 1
6960674 1
6964263 1
6964939 0
6971416 1
6971549 1
6974902 0
6994614 1
7003892 1
7013449 0
7019475 1
7049561 1
7058345 0
7061410 0
7066206 1
7079555 1
7086490 0
7087366 0
7098815 1
7108320 1
7121248 0
7131996 1
7133614 1
7142342 1
7173608 0
7180104 0
7191041 0
7212583 0
7222702 1
7224726 0
7226280 0
7239611 0
7254737 1
7260802 0
7274682 0
7279241 1
7290274 1
7320385 0
7337764 1
7340367 0
7342267 1
7343753 0
7354391 0
7361759 0
7369690 0
7370287 1
7383148 0
7399857 0
7402785 1
7404026 1
7408843 0
7419163 1
7432161 1
7452289 1
7489751 1
7513548 0
7528962 1
7546246 1
7569261 0
7583143 1
7591886 0
7619872 0
7715715 0
7741823 0
7747897 0
7751435 1
7755534 0
7759905 0
7784259 0
7793278 0
7809657 0
7822335 1
7827428 0
7828324 0
7832614 1
7849222 0
7855739 1
7858662 1
7891109 1
7893256 1
7893604 1
7907515 1
7908170 1
7914397 1
7918347 0
7950479 0
7962657 1
7964185 1
7977625 0
7981589 1
7995917 0
7997719 0
7998263 0
8034775 0
8040958 0
8061897 1
8073696 0
8084874 0
8087678 0
8088336 0
8091692 1
8092961 0
8108442 1
8119852 0
8123349 1
8181249 1
8196487 0
8206940 1
8213230 0
8243196 0
8245996 1
8248948 1
8253159 1
8259140 0
8262528 1
8277406 0
8280315 1
8295397 0
8300821 0
8305011 0
8305576 1
8309975 0
8319612 1
8337566 1
8365399 1
8374777 1
8380680 0
8408134 1
8415149 1
8421414 1
8422996 0
8438127 0
8470467 0
8489186 0
8490102 1
8490653 0
8516682 1
8523429 0
8529036 1
8536880 1
8585299 1
8653995 1
8663191 1
8676367 0
8708941 0
8738634 1
8741281 0
8750780 0
8751802 1
8754169 0
8782351 0
8784833 1
8791640 0
8792846 1
8807331 1
8814284 0
8827374 1
8835425 1
8838436 1
8878924 0
8879828 0
8897371 0
8901437 1
8902658 0
8919136 1
8927367 1
8940077 0
8951804 0
8983324 0
9015691 1
9035773 1
9052602 0
9059665 0
9063289 0
9071637 1
9081079 1
9091001 0
9113290 0
9118127 1
9123482 1
9130326 1
9130756 1
9140669 1
9162896 1
9165382 0
9176881 0
9221926 0
9223157 1
9230590 1
9248406 1
9249282 0
9263262 0
9264449 1
9267158 1
9289479 1
9289987 1
9294977 1
9312289 0
9337613 0
9360667 0
9363013 0
9377424 0
9387850 0
9388803 0
9412259 0
9413041 0
9416543 1
9454861 0
9456253 0
9467673 1
9472592 1
9507710 1
9515023 1
9521692 0
9527242 1
9537933 1
9573060 0
9591876 1
9595324 0
9623261 0
9623726 1
9632569 1
9632783 0
9645217 1
9660308 0
9674927 1
9683203 0
9693184 0
9699485 1
9700951 0
9702722 0
9708911 0
9713592 0
9717413 0
9732735 0
9761897 1
9773386 1
9780965 0
9785854 0
9790926 0
9798079 0
9801317 0
9814327 0
9828480 1
9829735 0
9839478 0
9849589 1
9872627 0
9878996 0
9887662 1
9940042 1
9942176 0
9952714 1
9964613 1
9974352 1
9982881 1
10005871 1
10007111 0
10011395 0
10043809 0
10047192 1
10047851 1
10052018 0
10059541 1
10070935 1
10078226 1
10082070 0
10084471 1
10108689 0
10113950 1
10137504 1
10141932 0
10147788 0
10162501 1
10171511 0
10182051 0
10219941 0
10226034 0
10226911 1
10241876 1
10258087 1
10291003 1
10299090 0
10312621 0
10317808 1
10318920 0
10320996 1
10323881 1
10326188 0
10336169 0
10345440 1
10367109 1
10376619 1
10396345 1
10399808 1
10419941 1
10423494 1
10426629 1
10429015 0
10432340 0
10437221 0
10438342 0
10455401 0
10458336 1
10469238 0
10480268 1
10516939 1
10523178 0
10523794 1
10543444 0
10547602 1
10549195 1
10553956 1
10565422 0
10569276 1
10579250 0
10590427 0
10600206 0
10619381 0
10621417 1
10624353 1
10646417 1
10652801 0
10669735 1
10672016 0
10679957 1
10682274 1
10697510 1
10703223 0
10711759 1
10712740 1
10713290 0
10725196 0
10738868 1
10741362 1
10749299 0
10768919 0
10769880 1
10770941 0
10771617 0
10785743 0
10810367 1
10837110 1
10839038 1
10845294 0
10866496 0
10881689 0
10892069 0
10894327 0
10899036 1
10903248 1
10905042 0
10936890 0
10940856 0
10948328 0
10968405 0
10975520 0
11021135 1
11040312 1
11041282 1
11045092 1
11069715 1
11089266 1
11090438 1
11092651 0
11103734 0
11123447 1
11128949 0
11136058 1
11185083 1
11188955 0
11206756 1
11207973 1
11212711 1
11219558 0
11230287 0
11276029 0
11283794 1
11291016 0
11295283 1
11296361 0
11298609 0
11314228 1
11319826 1
11363623 0
11371719 0
11373054 0
11401773 0
11419366 0
11421715 1
11440609 1
11448583 1
11448707 0
11460203 0
11470174 1
11475635 1
11483255 1
11485783 1
11494660 0
11497108 1
11499021 0
11536597 1
11544410 0
11588738 1
