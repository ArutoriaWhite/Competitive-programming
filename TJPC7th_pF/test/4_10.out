10975520 11559767
